liquidator/src/liquidate.rs

```diff
@@ -168,7 +168,7 @@ impl<'a> LiquidateHelper<'a> {
         > = stream::iter(self.liqee.active_perp_positions())
             .then(|pp| async {
                 let base_lots = pp.base_position_lots();
-                if base_lots == 0 {
+                if base_lots == 0 || pp.has_open_taker_fills() {
```

Comment:
so I am guessing thought here is that the user reduced his position, and that means liqor would take on even less liabs? He obviously can't increase if he is liquidatable

Reply:
Yeah, the thought is that reducing base_position to 0 is useless when the incoming fill is going to change it anyway. Possibly we could aim for bringing the effective_base_pos to 0, but it may be safer to just wait for the fill to happen - that will also change the user health, after all.